rpa-builder/6.0/modules/ROOT/pages/using-workflow-debugger.adoc

```diff
@@ -0,0 +1,57 @@
+= Using Workflow Debugger
+
+RPA Builder provides an integrated debugger for Workflows. To use it, click the *Debug Workflow* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) or press _F8_. This starts a test run and opens Workflow Debugger instead of stopping the execution of the Workflow if an error occurs.
```

Comment:
```suggestion
RPA Builder provides an integrated debugger for Workflows. 

To use the debugger, click *Debug Workflow* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) or press _F8_. This starts a test run and opens Workflow Debugger instead of stopping the execution of the Workflow if an error occurs.
```